apps/backend/src/environment.ts

```diff
@@ -7,5 +9,15 @@ export const environment = {
   assetPath: __dirname + '/assets/',
   polls: {
     activeDays: +process.env.ACTIVE_DAYS || 7, // how many days a poll is active after the deadline is reached
-  }
+  },
+  auth: {
+    publicKey: `-----BEGIN PUBLIC KEY-----\n${
+      // see http://localhost:8080/auth/admin/master/console/#/apollusia/realm-settings/keys/list -> RS256 -> Public Key
+      process.env.AUTH_PUBLIC_KEY || 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA6RcQMrMgYacLZV2S6HXuPfbo6dF4y8ZjqrkMEM+Nj2Jt96BC4Ss9O9EkhQonXy2FWcUxvmEKicRApAMXto7r6TiE/LCi+pLfMnAxcIsiQIqDZrVWn5y1jz8gdKuvPxsTW1q3XEjqWuYUZof3/jahBrr1BBtNYRMonOC50a7d8wg7IBxwC3kk9zh8MImfJw6BZmSQ8fNuoHCORsUht7HW1h+HtbFEQEcTVUPxXUvV7Mw6Bgm7oWJLA3gG3heuOyUapqKnGkkMaJrA2xGrW9/66I65UD8jZgoEv5n2uGwXsmAafxZfoA+Mx/xCY6zEhpbxedtp99M2k0pvitwZmXCsnwIDAQAB'
```

Comment:
Should be documented in README.md.

Reply:
Will do later along with more keycloak setup instructions.